### PR TITLE
iptables fixes

### PR DESCRIPTION
**Pull request by dspicuzzbbn**
This PR fixes two things:

* Allows not specifying the `:FORWARD ACCEPT [0:0]` style of defaults (iptables-save doesn't require it, so neither should batfish)
* Applies iptables rules without explicit interfaces specified to all interfaces, and adds a test for that

### CODE REVIEW

**Comment by dspicuzzbbn** [comment]
This partially resolves #485, but I believe that there are further issues that need to be investigated by someone else.

**Comment by dspicuzzbbn** [comment]
We've added another commit with a new iptables testrig that does more comprehensive tests.

File: tests/basic/example-iptables-protection.ref [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "answerElements" : [
+    {
+      "class" : "org.batfish.datamodel.FlowHistory"
+    }
```
Comment (arifogel) [inline tests/basic/example-iptables-protection.ref]: Is this supposed to be empty? What about the other reachability answers?

Comment (virtuald) [reply tests/basic/example-iptables-protection.ref]: @jkhourybbn?

Comment (jkhourybbn) [reply tests/basic/example-iptables-protection.ref]: Yes all three questions are supposed to be empty. A non empty result is a violation of the policy. The questions are crafted so they return no results.

**Pull request merged**